index.html

```diff
@@ -44,4 +44,113 @@ <h4>
+      <img src="images/Tags.png" alt="html tag">
+      <a href="https://github.com/" class="button">See Project</a>
+    </div>
+    
+    
+    <div class="flex-cards">
+      <img src="images/Snapshoot-Portfolio-b.png" alt="Snapshoot">
+      <h3>Tonic</h3>
+      <span>
+        Canopy
+        <img src="images/Counter.png" alt="circle"> Back End Dev
+        <img src="images/Counter.png" alt="circle"> 2015
+      </span>
+    
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img src="images/Tags.png" alt="html tag">
+      <a href="https://github.com/" class="button">See Project</a>
+    </div>
+    
+    
+    <div class="flex-cards">
+      <img src="images/Snapshoot-Portfolio-c.png" alt="Snapshoot">
+      <h3>Multi-post series</h3>
+      <span>
+        Canopy
+        <img src="images/Counter.png" alt="circle"> Back End Dev
+        <img src="images/Counter.png" alt="circle"> 2015
+      </span>
+    
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img src="images/Tags.png" alt="html tag">
+      <a href="https://github.com/" class="button">See Project</a>
+    </div>
+  </section>
+    
+  <!-- Mobile About Section -->
+  <section class="about">
+    <h2>About myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+    <h4>
+      <a href="https://github.com/">LET’S CONNECT</a>
+    </h4>
+        
+    <nav class="social-links">
+      <ul>
+        <li>
+          <a href="https://twitter.com/"><img src="images/twitter-logo.png" alt="twitter-logo"></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/stephanie-ugboaja-930a2a216/"><img src="images/Linkedin-icon.png" alt="linkedin-logo"></a>
+        </li>
+        <li>
+          <a href="#" target="_blank"><img src="images/m-logo.png" alt="m"></a>
+        </li>
+        <li>
+          <a href="https://github.com/"><img src="images/github-logo.png" alt="github-logo"></a>
+         </li>
+         <li>
+          <a href="#" target="_blank"><img src="images/peace-logo.png" alt="A hand with two fingers up"></a>
+        </li>
+      </ul>
+      <a href="https://github.com/" class="resume">Get My Resume</a>
+    </nav>
+    
+    <ul class="list">
+      <li>Languages
+        <img src="images/Union.png" alt="Union" class="lang-tag">
+      </li>
+      <li>
+        <img src="images/Language-List-crop-Copy.png" alt="programming Languages" class="languages"> <hr>
+      </li> 
+      <li>Frameworks 
+        <img src="images/Union.png" alt="Union" class="frame-tag"> <hr>
+      </li>
+      <li>Skills
+        <img src="images/Union.png" alt="Union" class="skills-tag"> <hr>
+      </li>
+    </ul>
+  </section>
+</body>
```

Comment:
- [ ] This section should have the margins (top, bottom, left, right) as defined in the design. Please see the screenshot below for the marked differences between your design and the template. The ` Get My Resume` button and the `Languages`, `Frameworks` and `Skills ` tabs are not aligned in your design. Please let's fix it. As already mentioned, following project's requirement is an important skill. 

Design Template
![template-design](https://user-images.githubusercontent.com/70150423/129769439-1fcac4bf-9ecf-4fb5-bbd5-09d3bdb973af.png)
Your design
![Screenshot 2021-08-17 at 16-59-22 Portfolio setup and mobile version skeleton](https://user-images.githubusercontent.com/70150423/129769156-f2cdb2d5-78d9-40cc-bf73-dfa41305f5d1.png)